skipruntime-ts/adapters/postgres/src/index.ts

```diff
@@ -230,17 +230,19 @@ FOR EACH ROW EXECUTE FUNCTION %I();`,
   }
 
   unsubscribe(instance: string): void {
-    this.client
-      .query(format("DROP FUNCTION IF EXISTS %I CASCADE;", instance))
-      .then(
-        () => this.open_instances.delete(instance),
-        (e: unknown) => {
-          console.error(
-            "Error unsubscribing from resource instance: " + instance,
-          );
-          throw e;
-        },
-      );
+    if (this.open_instances.has(instance))
```

Comment:
Do we want to throw a specific error in case the user attempts to `unsubscribe()` from an instance not in `open_instance`, as it would likely indicate a bug in user code (vs this package)?